src/main/scala/scorex/api/http/swagger/SwaggerDocService.scala

```diff
@@ -20,6 +26,40 @@ class SwaggerDocService(val actorSystem: ActorSystem, val materializer: ActorMat
     Some(License("MIT License", "https://github.com/virtualeconomy/vsys/blob/master/LICENSE"))
   )
 
+  override def generateSwaggerJson: String = {
+    try {
+      Json.pretty().writeValueAsString(customizedSwagger)
+    } catch {
+      case NonFatal(t) => {
+        log.error("Issue with creating swagger.json", t)
+        throw t
+      }
+    }
+  }
+
+  override def generateSwaggerYaml: String = {
+    try {
+      Yaml.pretty().writeValueAsString(customizedSwagger)
+    } catch {
+      case NonFatal(t) => {
+        log.error("Issue with creating swagger.yaml", t)
+        throw t
+      }
+    }
+  }
+
+  private val cumstomPathsConfig: Map[String, Boolean] = Map(
+    "/transactions/count"  -> settings.customApiSettings.transactionsApiSettings.addressTransactionCount,
+    "/transactions/list"   -> settings.customApiSettings.transactionsApiSettings.addressTransactionList,
+  )
+
+  private def customizedSwagger: Swagger = {
+    val swagger: Swagger = reader.read(apiClasses.asJava)
+    val fiteredPaths: Map[String, Path] = swagger.getPaths().asScala.toMap.filterKeys((path) => cumstomPathsConfig.get(path).getOrElse(true))
```

Comment:
`()` for `path` is useless

we can use `getOrElse(path, true)` to simplify the logic here.

Reply:
updated